wren-launcher/utils/docker.go

```diff
@@ -306,8 +310,35 @@ func CheckWrenAIStarted(url string) error {
 	defer resp.Body.Close()
 
 	if resp.StatusCode != 200 {
+		fmt.Println("WrenUI is not ready yet")
```

Comment:
I'm thinking printing this will cause too many lines of information printing on terminal. Maybe we can remove this one ?

Reply:
delivered in 38544ea2cca6541252f3c7c738bd28b0d0526e93